Automattic/wp-calypso
# Remove error-reporting module and loader from ETK

Related to https://github.com/Automattic/jetpack/pull/34158

## Proposed Changes

* This PR is a follow-up to https://github.com/Automattic/jetpack/pull/34158, which replicates the code for the `error-reporting`/Sentry client module in the `jetpack-mu-wpcom` package. This change removes the relevant code from ETK so we only have one code location moving forward.

## Testing Instructions



* Apply changes to your site
  * simple sites: Run `cd apps/editing-toolkit && yarn dev --sync`
  * atomic sites: Download the plugin via [here](https://teamcity.a8c.com/buildConfiguration/calypso_calypso_WPComPlugins_Build_Plugins/12843074?buildTab=artifacts#)
* Go to your site
* Go to the editor
* Make sure everything works well

## Pre-merge Checklist
